### PR TITLE
test(main): Remove all instances of main

### PR DESCRIPTION
**Pull request by Droogans**
JIRA: https://jira.rax.io/browse/FRMW-431

### LGTMs
- [x] Dev LGTM
- [x] QE/Dev LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
@Droogans, we should chat about this on Monday (3/21), but I feel like replacing `main` with `initialize()`(with an optional selector argument) is just _moving_ the code smell rather than _removing_ it.  

If we are to remove `main`, I would suggest _requiring a **mandatory** selector argument_ for `initialize()`. @parlarjb or @halkon, do either of you have any comments/suggestions?

**Comment by Droogans** [comment]
What would forcing a mandatory selector solve? I don't seem to understand what the smell is here.

File: src/components/rxCharacterCount/scripts/rxCharacterCount.page.js [hunk 1]
```diff
@@ -129,20 +129,5 @@ exports.rxCharacterCount = {
             get: function () { return rxCharacterCountElement; }
         };
         return Page.create(rxCharacterCount);
```
Comment (CITguy) [inline src/components/rxCharacterCount/scripts/rxCharacterCount.page.js]: This one should require selector argument, and documented as such.

File: src/components/rxCollapse/scripts/rxCollapse.page.js [hunk 2]
```diff
@@ -148,19 +148,5 @@ exports.rxCollapse = {
             get: function () { return rxCollapseElement; }
         };
         return Page.create(rxCollapse);
```
Comment (CITguy) [inline src/components/rxCollapse/scripts/rxCollapse.page.js]: This should require a selector argument, and documented as such.

File: src/components/rxMultiSelect/scripts/rxMultiSelect.page.js [hunk 3]
```diff
@@ -248,18 +248,5 @@ exports.rxMultiSelect = {
             get: function () { return rxMultiSelectElement; }
         };
         return Page.create(rxMultiSelect);
```
Comment (CITguy) [inline src/components/rxMultiSelect/scripts/rxMultiSelect.page.js]: This should require a selector argument, and documented as such.

File: src/components/rxPaginate/scripts/rxPaginate.page.js [hunk 4]
```diff
@@ -537,19 +537,5 @@ exports.rxPaginate = {
             get: function () { return rxPaginationElement; }
         };
         return Page.create(rxPaginate);
```
Comment (CITguy) [inline src/components/rxPaginate/scripts/rxPaginate.page.js]: This should require a selector argument, and documented as such.

File: src/components/rxSearchBox/scripts/rxSearchBox.page.js [hunk 5]
```diff
@@ -155,19 +155,5 @@ exports.rxSearchBox = {
             get: function () { return rxSearchBoxElement; }
         };
         return Page.create(rxSearchBox);
```
Comment (CITguy) [inline src/components/rxSearchBox/scripts/rxSearchBox.page.js]: This should require a selector argument, and documented as such.

File: src/components/rxToggleSwitch/scripts/rxToggleSwitch.page.js [hunk 6]
```diff
@@ -139,18 +139,5 @@ exports.rxToggleSwitch = {
             get: function () { return rxToggleSwitchElement; }
         };
         return Page.create(rxToggleSwitch);
```
Comment (CITguy) [inline src/components/rxToggleSwitch/scripts/rxToggleSwitch.page.js]: This should require a selector argument, and documented as such.

**Comment by Droogans** [comment]
These requirements conflict with https://github.com/rackerlabs/encore-ui/pull/1602 (which is closed anyway, but still very much alive in spirit).

I'll get these done first and re-do the first PR to force this one as the source of truth for conflicts.

File: src/components/rxCharacterCount/scripts/rxCharacterCount.page.js [hunk 7]
```diff
@@ -120,7 +120,7 @@ exports.rxCharacterCount = {
      * @function
      * @memberof rxCharacterCount
      * @description Creates a page object from an `[rx-bulk-select]` DOM element.
-     * @param {ElementFinder} [rxCharacterCountElement=$('[rx-character-count]')] -
+     * @param {ElementFinder} rxCharacterCountElement -
```
Comment (Droogans) [inline src/components/rxCharacterCount/scripts/rxCharacterCount.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57070416.

File: src/components/rxCollapse/scripts/rxCollapse.page.js [hunk 8]
```diff
@@ -135,32 +135,14 @@ exports.rxCollapse = {
      * @function
      * @memberof rxCollapse
      * @description Creates a page object from an `[rx-collapse]` DOM element.
-     * @param {ElementFinder} [rxCollapseElement=$('rx-collapse')] -
+     * @param {ElementFinder} rxCollapseElement -
```
Comment (Droogans) [inline src/components/rxCollapse/scripts/rxCollapse.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57070575.

File: src/components/rxMultiSelect/scripts/rxMultiSelect.page.js [hunk 9]
```diff
@@ -240,26 +240,9 @@ exports.rxMultiSelect = {
      * @returns {rxMultiSelect}
      */
     initialize: function (rxMultiSelectElement) {
-        if (rxMultiSelectElement === undefined) {
-            rxMultiSelectElement = $('rx-multi-select');
-        }
-
```
Comment (Droogans) [inline src/components/rxMultiSelect/scripts/rxMultiSelect.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57070694.

File: src/components/rxPaginate/scripts/rxPaginate.page.js [hunk 10]
```diff
@@ -528,7 +528,7 @@ exports.rxPaginate = {
 
     /**
      * @function
-     * @param {ElementFinder} [rxPaginateElement=$('.rx-paginate')] - DOM element representing the rxPaginate component.
+     * @param {ElementFinder} rxPaginateElement - DOM element representing the rxPaginate component.
```
Comment (Droogans) [inline src/components/rxPaginate/scripts/rxPaginate.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57071012.

File: src/components/rxToggleSwitch/scripts/rxToggleSwitch.page.js [hunk 11]
```diff
@@ -126,31 +126,14 @@ exports.rxToggleSwitch = {
...
-     * @param {ElementFinder} [rxToggleSwitchElement=$('rx-toggle-switch')] -
+     * @param {ElementFinder} rxToggleSwitchElement -
      * ElementFinder to be transformed into an rxToggleSwitchElement object.
      * @returns {rxToggleSwitch} Page object representing the {@link rxToggleSwitch} object.
      */
     initialize: function (rxToggleSwitchElement) {
-        if (rxToggleSwitchElement === undefined) {
-            rxToggleSwitchElement = $('rx-toggle-switch');
-        }
-
```
Comment (Droogans) [inline src/components/rxToggleSwitch/scripts/rxToggleSwitch.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57071523.

File: src/components/rxSearchBox/scripts/rxSearchBox.page.js [hunk 12]
```diff
@@ -147,27 +147,9 @@ exports.rxSearchBox = {
      * @description Page object representing the rxSearchBox object.
      */
     initialize: function (rxSearchBoxElement) {
-        if (rxSearchBoxElement === undefined) {
-            rxSearchBoxElement = $('rx-search-box');
-        }
-
```
Comment (Droogans) [inline src/components/rxSearchBox/scripts/rxSearchBox.page.js]: Addressed https://github.com/rackerlabs/encore-ui/pull/1601/files#r57071079.

**Comment by CITguy** [comment]
LGTM

**Comment by CITguy** [comment]
Who else LGTM'ed this?

**Comment by Droogans** [comment]
Wiki updated: https://github.com/rackerlabs/encore-ui/wiki/2.0-Migration-Guide#2016-03-25

**Comment by 100stacks** [comment]
LGTM

**Comment by 100stacks** [comment]
@Droogans, might help to update from `main` to `initialize()` also here: https://github.com/rackerlabs/encore-ui/blob/master/guides/testing.md

**Comment by Droogans** [comment]
@100stacks https://github.com/rackerlabs/encore-ui/pull/1607

**Pull request merged**